tests/helpers.py

```diff
@@ -2101,3 +2101,18 @@ def modify_osd_replica_count(resource_name, replica_count):
     params = f'{{"spec": {{"replicas": {replica_count}}}}}'
     resource_name = '-'.join(resource_name.split('-')[0:4])
     return ocp_obj.patch(resource_name=resource_name, params=params)
+
+def checkLSO():
```

Comment:
function name with lower case and '_' as a separation between words --> check_LSO